ircd/s_serv.c

```diff
@@ -522,11 +525,14 @@ burst_modes_TS6(struct Client *client_p, struct Channel *chptr,
 		banptr = ptr->data;
 
 		if (banptr->forward)
-			send_multiline_item(client_p, "%s$%s",
-					banptr->banstr,
-					banptr->forward);
+			sprintf(buf, "%s$%s", banptr->banstr, banptr->forward);
 		else
-			send_multiline_item(client_p, "%s", banptr->banstr);
+			strcpy(buf, banptr->banstr);
+
+		if IsCapable(client_p, CAP_EBMASK)
+			sprintf(buf, "%s,%ld,%s", buf, (long)banptr->when, banptr->who);
```

Comment:
Making `sprintf()` copy between overlapping objects (`buf`) is UB. Perhaps it's better to use two `send_multiline_item` calls, one for traditional and one for EBMASK.